src/WorkBC.Data/SsotContext.cs

```diff
@@ -0,0 +1,208 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using WorkBC.Data.Model.Enterprise;
+
+namespace WorkBC.Data
+{
+    public partial class SsotContext : DbContext
+    {
+        private string ssotConnectionString;
+        
+        public SsotContext(DbContextOptions<SsotContext> options, IConfiguration configuration) 
+            : base(options)
+        {
+            ssotConnectionString = configuration.GetConnectionString("SsotConnection");
+        }
+
+        public virtual DbSet<CareerProfile> CareerProfiles { get; set; }
+        public virtual DbSet<IndustryProfile> IndustryProfiles { get; set; }
+        public virtual DbSet<Noc> Nocs { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseNpgsql(ssotConnectionString);
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<CareerProfile>(entity =>
+            {
+                entity.HasKey(e => e.CareerProfileId);
+
+                entity.ToTable("edm_careerprofile");
```

Comment:
Postgres table and column names are case-insensitive so every table name, will need to be modified here to show the new Postgres table name.